apps/climatemappedafrica/src/components/Navigation/ExploreNavigation/index.js

```diff
@@ -83,24 +89,26 @@ function ExploreNavigation({ explorePagePath, locations, logo, variant }) {
                 menuItem: classes.searchMenuItem,
               }}
             />
-            <Typography
-              component="div"
-              id="nav-help"
-              onClick={openTooltip}
-              variant="h3"
-              sx={(theme) => ({
-                color: "#666666",
-                textAlign: "center",
-                backgroundColor: "#EBEBEB",
-                borderRadius: theme.typography.pxToRem(60),
-                marginLeft: theme.typography.pxToRem(20),
-                width: theme.typography.pxToRem(48),
-                height: theme.typography.pxToRem(48),
-                cursor: "pointer",
-              })}
-            >
-              ?
-            </Typography>
+            {tutorialEnabled && (
+              <Typography
+                component="div"
+                id="nav-help"
+                onClick={openTooltip}
```

Comment:
If it's something that needs to be clicked, why aren't we using a `Button` or `Link`?